### ISSUE TITLE
Package flatpack

### ISSUE THREAD

**Issue by mudler** [mocaccinoOS/issue-tracker#23]
We should package flatpack for portage-tree at least, and explore if it's feasable in mocaccino-musl-universe

**Comment by jcfrosty** [comment]
Yes! flatpak would be an excellent addition and a massive win for end-users!

**Comment by geaaru** [comment]
https://github.com/mocaccinoOS/portage-tree/blob/master/multi-arch/packages/groups/collection.yaml#L257

added gid entity

**Comment by joostruis** [comment]
https://github.com/mocaccinoOS/desktop/tree/master/packages/apps/flatpak
Works on desktop
